tests/validators.py

```diff
@@ -474,3 +474,44 @@ def validate_rook_ceph():
     """
     wait_for_installation()
     wait_for_pod_state("", "rook-ceph", "running", label="app=rook-ceph-operator")
+
+
+def validate_rook_ceph_integration():
+    """
+    Integration test for rook-ceph microceph.
+    """
+    wait_for_installation()
+    wait_for_pod_state("", "rook-ceph", "running", label="app=rook-ceph-operator")
+
+    manifest = TEMPLATES / "microceph.yaml"
+    try:
+        subprocess.check_call("microceph cluster bootstrap".split())
+        subprocess.check_call("microceph status".split())
+
+        # Prepare devices
+        for l in ["a", "b", "c"]:
+            if not os.path.exists(f"/dev/sdi{l}"):
+                loop_file = subprocess.check_output("mktemp -p /mnt XXXX.img".split())
+                loop_file = loop_file.decode().strip()
+                subprocess.check_call(f"truncate -s 1G {loop_file}".split())
+                loop_dev = subprocess.check_output(
+                    f"losetup --show -f {loop_file}".split()
+                )
+                loop_dev = loop_dev.decode().strip()
+                minor = loop_dev.replace("/dev/loop", "")
+                subprocess.check_call(f"mknod -m 0660 /dev/sdi{l} b 7 {minor}".split())
+
+            subprocess.check_call(f"microceph disk add --wipe /dev/sdi{l}".split())
+
+        subprocess.check_call("ceph fs volume create fs0".split())
+        subprocess.check_call("microk8s connect-external-ceph".split())
+
+        kubectl("apply -f {}".format(manifest))
+        wait_for_pod_state("nginx-rbd", "default", "running")
+        # We do not test ceph-fs because its provisioner requires CPU cores
+        # that may not be avaibale on small VMs.
+        # wait_for_pod_state("nginx-fs-1", "default", "running")
+        # wait_for_pod_state("nginx-fs-2", "default", "running")
```

Comment:
Do we need those comments?

Reply:
Lets keep this comment so that if we want to test ceph-fs on a larger machine we can comment in these two lines instead of trying to figure out then how to update the test.